### ISSUE TITLE
[Enhancement] Torrent config option: Default download path

### ISSUE THREAD

**Issue by marcelpaulo** [SchizoDuckie/DuckieTV#1103]
**What build of DuckieTV are you using (Standlone / Chrome Extension (New Tab / Browser Action))**
Standalone

**What version of DuckieTV are you using (Stable 1.1.x / Nightly yyyymmddHHMM)**
git angular (https://github.com/SchizoDuckie/DuckieTV/commit/4866616226e525c35a37f8ff7237d18332283e95)

**What is your Operating System (Windows, Mac, Linux, Android)**
Linux (Xubuntu 18.04)

**Describe the problem you are having and steps to reproduce if available**
It would be very useful to have a torrent config option **default download path**. I know the download path can be configured for each series. But it would be useful to be able to set a default download path for all series, instead of setting it individually for each series.

**Comment by garfield69** [comment]
I don't see the need.
The default download path is set in your torrent client.
Those using the dlpath for a series usually intend to put each series in a separate folder.

**Comment by marcelpaulo** [comment]
I put all series in the same folder, where I run subliminal every hour to download subtitles. It's not the global transmission default folder.

Right now, I have to configure each series to this same download path.

I would imagine this is a common scenario. A single folder for each type of download: one for series, one for movies, one for music, ... Downloading subtitles for movies and episodes optimally use different scores, hence the need to partition them with in different folders.

**Comment by garfield69** [comment]
Most users that download tv series put each series in their own folder, rather than dump every series into one folder. You are the first to ask for a common folder for all the series that they are using duckietv to track.
But I'll leave the ticket open for a while and see if there is any additional interest for this.

**Comment by marcelpaulo** [comment]
I used to dump all videos in the same folder, but then I couldn't properly automate downloading subtitles with subliminal: it gets better results using a different `--min-score` for movies and episodes. So I set a folder for each one.

Now that I'm thinking about it, it would also work with each series in its folder, as long as they were all subfolders of a same folder. I think I finally see your point. But I still prefer a single folder.

Of course, it's no big deal to set up the download path for each series added to the favorites. Actually I didn't even know that was possible, so I was trying to set up a scheme to choose the download path for each individual **episode**. Pre-configuring the folder in DuckieTV for each series is already a huge life saver. 

Let's do it differently: I'll close the issue, and if someone shows interest, which seems unlikely as you explained, it may be reopened.
